pkg/runtime/embedded.go

```diff
@@ -24,9 +24,9 @@ import (
 )
 
 type encodable struct {
-	e        Encoder `json:"-"`
+	E        Encoder `json:"-"`
 	obj      Object
-	versions []unversioned.GroupVersion `json:"-"`
+	Versions []unversioned.GroupVersion `json:"-"`
```

Comment:
This change is curious. Was vet complaining about json tags on private members? I think if that's the case, remove the tags and leave the members private.